### PR TITLE
doc: document metrics exported by CephFS

### PR DESCRIPTION
**Pull request by joscollin**
Fixes: https://tracker.ceph.com/issues/66173

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins render docs

**Review by zdover23 — APPROVED** [review]

**Comment by joscollin** [comment]
@vshankar `admin/build-doc` build is successful and the generated doc looks good. But the Jenkins `docs/readthedocs.org:ceph` fails and it's unrelated.

**Comment by anthonyeleven** [comment]
Question:  these metrics are harvested from where?  The admin socket of any active MDS?  The admin socket of the MDS for a given rank?  Something else?  I think it would be valuable to briefly note this.

**Comment by joscollin** [comment]
> Question: these metrics are harvested from where? The admin socket of any active MDS? The admin socket of the MDS for a given rank? Something else? I think it would be valuable to briefly note this.

Updated

File: doc/cephfs/metrics.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+     - Total number of write operations generated by all process
+   * - total_write_size
+     - Gauge
+     - Number of bytes written in input/output operations generated by all processes
+
+Getting Metrics
+===============
+
+The metrics could be scraped from the MDS admin socket as well as using the tell interface. The ``mds_client_metrics-<fsname>`` section in the output of ``counter dump`` command displays the metrics for each client as shown below::
+
```
Comment (neesingh-rh) [inline doc/cephfs/metrics.rst:73]: Just an opinion: If we add a sample command for counter dump, would be better.

Comment (joscollin) [reply doc/cephfs/metrics.rst:73]: We already have reference to `Perf Counters` doc, which has the command samples.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/57723 - Squid backport
https://github.com/ceph/ceph/pull/57724 - Reef backport
No Quincy backport

**Pull request merged**